lib/swoosh/adapters/mailgun.ex

```diff
@@ -86,13 +88,17 @@ defmodule Swoosh.Adapters.Mailgun do
 
   defp prepare_attachments(body, %{attachments: []}), do: body
   defp prepare_attachments(body, %{attachments: attachments}) do
-    Map.put(body, :attachments, Enum.map(attachments, &prepare_file(&1)))
+    {normal_attachments, inline_attachments} = Enum.split_with(attachments, fn(%Swoosh.Attachment{type: type}) -> type == :attachment end)
```

Comment:
separate lines
```
{..} =
  ...
```

Reply:
Remove `Swoosh.Attachment` and unnecessary braces, `fn %{...} -> ... end is sufficient`